src/globus_cli/commands/group/list.py

```diff
@@ -0,0 +1,23 @@
+from globus_cli.login_manager import LoginManager
+from globus_cli.parsing import command
+from globus_cli.services.groups import get_groups_client
+from globus_cli.termio import formatted_print
+
+
+@command("list", short_help="List groups you belong to")
+@LoginManager.requires_login(LoginManager.GROUPS_RS)
+def group_list():
+    """List all groups for the current user"""
+    client = get_groups_client()
+
+    groups = client.get_my_groups()
+
+    formatted_print(
+        groups,
+        fields=[
+            ("Group ID", "id"),
+            ("Name", "name"),
+            ("Type", "group_type"),
+            ("High Assurance", "enforce_session"),
```

Comment:
Although I would think `High Assurance` is a fine/good name in isolation, after looking at the webapp I think we should change this to `Session Enforcement` with values of `strict` and `not strict`. I think that's just
```python
lambda x: "strict" if x["enforce_session"] else "not strict"
```

Here's how the webapp presents this info to a group admin:
![image](https://user-images.githubusercontent.com/1300022/133528306-4bffede1-54c9-40fa-8b98-b0fb9ce4c73c.png)


While I'd love for us to be consistent across services and call this `High Assurance` _everywhere_, we can't be consistent both with the webapp and with the GCP/GCS context. Given that choice, I think that using a different name for this in the CLI vs webapp will create more confusion than having a different name between Groups and Collections.